examples/step-42/step-42.cc

```diff
@@ -152,14 +152,11 @@ namespace Step42
                                         double nu,
                                         double sigma_0,
                                         double gamma)
-    : kappa(E / (3 * (1 - 2 * nu)))
-    , mu(E / (2 * (1 + nu)))
-    , sigma_0(sigma_0)
-    , gamma(gamma)
-    , stress_strain_tensor_kappa(kappa *
-                                 outer_product(unit_symmetric_tensor<dim>(),
-                                               unit_symmetric_tensor<dim>()))
-    , stress_strain_tensor_mu(
+    : kappa(E / (3 * (1 - 2 * nu))), mu(E / (2 * (1 + nu))), sigma_0(sigma_0),
+      gamma(gamma), stress_strain_tensor_kappa(
```

Comment:
I veto this monstrosity. clang-format is bad enough, but the previous version at least put each variable on its own line. Here, it starts the second variable on the same line just to break the initializer onto the next line. This makes no sense.